net/acme/Makefile

```diff
@@ -8,17 +8,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=acme
-PKG_SOURCE_VERSION:=db3264ab8c8ee1aee1a84407702db4375f67765b
-PKG_VERSION:=1.6
-PKG_RELEASE:=4
+PKG_VERSION:=2.77
+PKG_RELEASE:=1
 PKG_LICENSE:=GPLv3
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_RELEASE).tar.xz
-PKG_MIRROR_HASH:=a881b86fef2b754739fc0024efdf061312fecaf613502434770b18191bda741d
-PKG_SOURCE_URL:=git://github.com/Neilpang/acme.sh.git
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_RELEASE)
-PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)-$(PKG_RELEASE)
+PKG_SOURCE_URL:=https://github.com/Neilpang/acme.sh
+PKG_SOURCE_VERSION:=8ab8a6eefbaf70e9bfa9fa30ab67f2c6a0bd1268
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
```

Comment:
This should still include $PKG_RELEASE.

Reply:
PKG_RELEASE was removed earlier.

Reply:
Ah yes, so it was. Well, it shouldn't be; otherwise we have no way to bump the version for changes in the packaging that do not come from upstream...

Reply:
Oh NVM. I just changed it to 1. If you're talking about the source subdir, there's no point. The build system is smart enough to figure out that the build directory needs updating.

PKG_SOURCE* does not impact the filename from opkg that users install. That's my understanding anyway.

Reply:
Hmm, it may be that the build system has changed since then, but I've had issues before with cached versions not being updated because the release was not in the filename. Will see if I can dig up the old example tomorrow...